src/GitHub/Endpoints/Organizations/Teams.hs

```diff
@@ -39,6 +39,7 @@ module GitHub.Endpoints.Organizations.Teams (
 import GitHub.Data
 import GitHub.Internal.Prelude
 import GitHub.Request
+import qualified Network.HTTP.Types as W
```

Comment:
We use `stylish-haskell`, to make import pretty put `qualified` imports into own section:

```haskell
import Github.Request
import Prelude ()

import qualified Network.HTTP.Types as W
```